submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/model/RequestStatus.kt

```diff
@@ -1,8 +1,16 @@
 package ac.uk.ebi.biostd.persistence.common.model
 
 import ebi.ac.uk.extended.model.ExtSubmission
+import java.time.OffsetDateTime
 
-data class SubmissionRequest(val submission: ExtSubmission, val draftKey: String?, val status: RequestStatus)
+data class SubmissionRequest(
+    val submission: ExtSubmission,
+    val draftKey: String?,
+    val status: RequestStatus,
+    val totalFiles: Int = 0,
```

Comment:
rather than help IMO I think this default parameters (exception modificationTime one which IMO is safe to have default value) may allow us to create a request without calculating this fields which will generate an issue for its proccessing. Basically this allow to easily introduce a bug.